content/docs/user-guide/components/reference/physx/prismatic-joint.md

```diff
@@ -0,0 +1,65 @@
+---
+linkTitle: PhysX Prismatic Joint
+description: ' Lean to use the Open 3D Engine PhysX Prismatic Joint component. '
+title: PhysX Prismatic Joint Component
+---
+
+
+
+With the **PhysX Prismatic Joint** component, you can create a dynamic prismatic joint that constrains an entity to the joint keeping the same rotation but allowing to move freely along one axis.
```

Comment:
nit: but allowing to move freely along one axis. -> but allowing it to move freely along one axis.

Reply:
Thanks, will change.